src/sentry/migrations/0024_auto_20191230_2052.py

```diff
@@ -74,8 +74,8 @@ def _attach_related(_events):
             project_id=e.project_id, event_id=e.event_id, group_id=e.group_id, data=e.data.data
         )
         primary_hash = event.get_primary_hash()
-        if event.project is None or event.group is None:
-            print("Skipped {} as group or project information is invalid.\n".format(event))
+        if event.project is None or event.group is None or len(event.data) == 0:
```

Comment:
Would it be safer to check for `not event.data`?

Reply:
I don't think that works since event.data is an instance of NodeData and has a bunch of other properties there, the len method is overloaded to give us what we are concerned with here.